### ISSUE TITLE
Hands on Training Accessibility Testing Workshop Webinar

### ISSUE THREAD

**Issue by teleject** [a11yproject/a11yproject.com#884]
April 2, 2020

Ensuring accessibility into our sites and apps should be seen as a challenge rather than a chore. By working on real life sites, as well as some code snippets, this workshop will demonstrate an accessibility testing workflow, which can be integrated into your own coding workflow. Also, we will include a review of automated testing, and hands-on manual testing using a variety of tools, from using the keyboard to using a screen reader application.

https://knowbility.org/services/training/hands-on-training-accessibility-testing-workshop/

**Comment by ericwbailey** [comment]
Hey @teleject, mind updating this to use the event issue template?

**Comment by teleject** [comment]
@ericwbailey , don't mind. I didn't know there was one!
